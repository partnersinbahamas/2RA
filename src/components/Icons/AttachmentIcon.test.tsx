import React from 'react';
import { render } from '@testing-library/react';
import AttachmentIcon from './AttachmentIcon';

describe('AttachmentIcon', () => {
  it('renders small', () => {
    const { container } = render(<AttachmentIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<AttachmentIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<AttachmentIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<AttachmentIcon />);
    expect(container).toMatchSnapshot();
  });
});
