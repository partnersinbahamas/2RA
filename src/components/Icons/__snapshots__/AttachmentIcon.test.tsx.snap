// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AttachmentIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="1 0.5 26 28"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0_1306_3534)"
    >
      <path
        clip-rule="evenodd"
        d="M23.8284 3.65685C22.2663 2.09476 19.7337 2.09476 18.1716 3.65685L6.15076 15.6777C5.76023 16.0682 5.12707 16.0682 4.73654 15.6777C4.34602 15.2871 4.34602 14.654 4.73654 14.2635L16.7574 2.24264C19.1005 -0.100505 22.8995 -0.100505 25.2426 2.24264C27.5858 4.58579 27.5858 8.38478 25.2426 10.7279L8.62563 27.3449C6.86827 29.1023 4.01903 29.1023 2.26167 27.3449C0.504311 25.5876 0.504312 22.7383 2.26167 20.981L13.9289 9.31371C15.1005 8.14213 17 8.14214 18.1716 9.31371C19.3431 10.4853 19.3431 12.3848 18.1716 13.5563L11.1005 20.6274C10.71 21.0179 10.0768 21.0179 9.68629 20.6274C9.29577 20.2369 9.29577 19.6037 9.68629 19.2132L16.7574 12.1421C17.1479 11.7516 17.1479 11.1184 16.7574 10.7279C16.3668 10.3374 15.7337 10.3374 15.3431 10.7279L3.67588 22.3952C2.69957 23.3715 2.69957 24.9544 3.67588 25.9307C4.6522 26.907 6.23511 26.907 7.21142 25.9307L23.8284 9.31371C25.3905 7.75161 25.3905 5.21895 23.8284 3.65685Z"
        fill-rule="evenodd"
      />
    </g>
    <defs>
      <clippath
        id="clip0_1306_3534"
      >
        <rect
          fill="white"
          height="29"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</div>
`;

exports[`AttachmentIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="1 0.5 26 28"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0_1306_3534)"
    >
      <path
        clip-rule="evenodd"
        d="M23.8284 3.65685C22.2663 2.09476 19.7337 2.09476 18.1716 3.65685L6.15076 15.6777C5.76023 16.0682 5.12707 16.0682 4.73654 15.6777C4.34602 15.2871 4.34602 14.654 4.73654 14.2635L16.7574 2.24264C19.1005 -0.100505 22.8995 -0.100505 25.2426 2.24264C27.5858 4.58579 27.5858 8.38478 25.2426 10.7279L8.62563 27.3449C6.86827 29.1023 4.01903 29.1023 2.26167 27.3449C0.504311 25.5876 0.504312 22.7383 2.26167 20.981L13.9289 9.31371C15.1005 8.14213 17 8.14214 18.1716 9.31371C19.3431 10.4853 19.3431 12.3848 18.1716 13.5563L11.1005 20.6274C10.71 21.0179 10.0768 21.0179 9.68629 20.6274C9.29577 20.2369 9.29577 19.6037 9.68629 19.2132L16.7574 12.1421C17.1479 11.7516 17.1479 11.1184 16.7574 10.7279C16.3668 10.3374 15.7337 10.3374 15.3431 10.7279L3.67588 22.3952C2.69957 23.3715 2.69957 24.9544 3.67588 25.9307C4.6522 26.907 6.23511 26.907 7.21142 25.9307L23.8284 9.31371C25.3905 7.75161 25.3905 5.21895 23.8284 3.65685Z"
        fill-rule="evenodd"
      />
    </g>
    <defs>
      <clippath
        id="clip0_1306_3534"
      >
        <rect
          fill="white"
          height="29"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</div>
`;

exports[`AttachmentIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="1 0.5 19 21"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0_1306_3532)"
    >
      <path
        clip-rule="evenodd"
        d="M16.8284 3.65685C15.2663 2.09476 12.7337 2.09476 11.1716 3.65685L4.10051 10.7279C3.70998 11.1184 3.07682 11.1184 2.68629 10.7279C2.29577 10.3374 2.29577 9.70423 2.68629 9.31371L9.75736 2.24264C12.1005 -0.100505 15.8995 -0.100505 18.2426 2.24264C20.5858 4.58579 20.5858 8.38478 18.2426 10.7279L8.6967 20.2739C6.93934 22.0312 4.0901 22.0312 2.33274 20.2739C0.575379 18.5165 0.575379 15.6673 2.33274 13.9099L9.05025 7.19239C10.2218 6.02081 12.1213 6.02082 13.2929 7.19239C14.4645 8.36396 14.4645 10.2635 13.2929 11.435L9.05025 15.6777C8.65973 16.0682 8.02656 16.0682 7.63604 15.6777C7.24551 15.2871 7.24552 14.654 7.63604 14.2635L11.8787 10.0208C12.2692 9.63029 12.2692 8.99713 11.8787 8.6066C11.4882 8.21608 10.855 8.21608 10.4645 8.6066L3.74695 15.3241C2.77064 16.3004 2.77064 17.8833 3.74695 18.8596C4.72326 19.836 6.30618 19.836 7.28249 18.8596L16.8284 9.31371C18.3905 7.75161 18.3905 5.21895 16.8284 3.65685Z"
        fill-rule="evenodd"
      />
    </g>
    <defs>
      <clippath
        id="clip0_1306_3532"
      >
        <rect
          fill="white"
          height="22"
          width="20"
        />
      </clippath>
    </defs>
  </svg>
</div>
`;

exports[`AttachmentIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="1 0.5 16.7 19"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0_1306_3530)"
    >
      <path
        clip-rule="evenodd"
        d="M14.6775 3.65685C13.1155 2.09476 10.5828 2.09476 9.02069 3.65685L3.36384 9.31371C2.97332 9.70423 2.34015 9.70423 1.94963 9.31371C1.5591 8.92318 1.5591 8.29002 1.94963 7.89949L7.60648 2.24264C9.94963 -0.100505 13.7486 -0.100505 16.0918 2.24264C18.4349 4.58579 18.4349 8.38478 16.0918 10.7279L8.66714 18.1525C6.90978 19.9099 4.06054 19.9099 2.30318 18.1525C0.54582 16.3952 0.545821 13.5459 2.30318 11.7886L7.60648 6.48528C8.77805 5.31371 10.6775 5.31371 11.8491 6.48528C13.0207 7.65685 13.0207 9.55635 11.8491 10.7279L8.31359 14.2635C7.92306 14.654 7.2899 14.654 6.89937 14.2635C6.50885 13.8729 6.50885 13.2398 6.89937 12.8492L10.4349 9.31371C10.8254 8.92318 10.8254 8.29002 10.4349 7.89949C10.0444 7.50897 9.41122 7.50897 9.02069 7.89949L3.71739 13.2028C2.74108 14.1791 2.74108 15.762 3.71739 16.7383C4.6937 17.7146 6.27662 17.7146 7.25293 16.7383L14.6775 9.31371C16.2396 7.75161 16.2396 5.21895 14.6775 3.65685Z"
        fill-rule="evenodd"
      />
    </g>
    <defs>
      <clippath
        id="clip0_1306_3530"
      >
        <rect
          fill="white"
          height="20"
          width="18"
        />
      </clippath>
    </defs>
  </svg>
</div>
`;
